src/recon_buildblock/BinNormalisationFromProjData.cxx

```diff
@@ -141,23 +141,21 @@ void
 BinNormalisationFromProjData::apply(RelatedViewgrams<float>& viewgrams,const double start_time, const double end_time) const 
   {
     this->check(*viewgrams.get_proj_data_info_sptr());
-    const ViewSegmentNumbers vs_num=viewgrams.get_basic_view_segment_num();
+    const ViewSegmentTOFNumbers vs_num=viewgrams.get_basic_view_segment_num();
 	const int timing_pos_num = norm_proj_data_ptr->get_proj_data_info_sptr()->is_tof_data() ? viewgrams.get_basic_timing_pos_num() : 0;
     shared_ptr<DataSymmetriesForViewSegmentNumbers> symmetries_sptr(viewgrams.get_symmetries_ptr()->clone());
-    viewgrams *= 
-      norm_proj_data_ptr->get_related_viewgrams(vs_num,symmetries_sptr, false,timing_pos_num);
+    viewgrams *= norm_proj_data_ptr->get_related_viewgrams(vs_num,symmetries_sptr, false);
   }
 
 void 
 BinNormalisationFromProjData::
 undo(RelatedViewgrams<float>& viewgrams,const double start_time, const double end_time) const 
   {
     this->check(*viewgrams.get_proj_data_info_sptr());
-    const ViewSegmentNumbers vs_num=viewgrams.get_basic_view_segment_num();
+    const ViewSegmentTOFNumbers vs_num=viewgrams.get_basic_view_segment_num();
 	const int timing_pos_num = norm_proj_data_ptr->get_proj_data_info_sptr()->is_tof_data() ? viewgrams.get_basic_timing_pos_num() : 0;
     shared_ptr<DataSymmetriesForViewSegmentNumbers> symmetries_sptr(viewgrams.get_symmetries_ptr()->clone());
-    viewgrams /= 
-      norm_proj_data_ptr->get_related_viewgrams(vs_num,symmetries_sptr, false, timing_pos_num);
+    viewgrams /= norm_proj_data_ptr->get_related_viewgrams(vs_num,symmetries_sptr, false);
```

Comment:
Remove `false` and what does the `/=` do?

Reply:
`./=` divides